qiskit_dynamics/pulse/pulse_to_signals.py

```diff
@@ -119,29 +125,33 @@ def get_signals(self, schedule: Schedule) -> List[DiscreteSignal]:
                 if isinstance(inst.pulse, Waveform):
                     inst_samples = inst.pulse.samples
                 else:
```

Comment:
If `isinstance(inst.pulse, Waveform) == False`, is the only other option for `inst.pulse` to be a `SymbolicPulse`? Do we need to worry about any other cases in this code block?